packages/table/components/Column.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+
+export interface IColumnProps {
+  label: string | JSX.Element;
+  cellRenderer: (data: any) => JSX.Element;
+  width: (
+    args: {
+      width: number;
+      totalColumns: number;
+      remainingWidth: number;
+      currentIndex: number;
+    }
+  ) => number;
+}
+
+export class Column extends React.PureComponent<IColumnProps, {}> {}
```

Comment:
This seems pretty unfinished, is a column really an empty component?

Reply:
Yes it is. It is not intended to render.